PiwikTracker/URLSessionDispatcher.swift

```diff
@@ -92,6 +92,10 @@ fileprivate extension Event {
                 URLQueryItem(name: "m", value: DateFormatter.minuteDateFormatter.string(from: date)),
                 URLQueryItem(name: "s", value: DateFormatter.secondsDateFormatter.string(from: date)),
                 
+                
+                //screen resolution
+                URLQueryItem(name: "res", value: String(describing: screenResolution.width)+"x"+String(describing: screenResolution.height)),
```

Comment:
What do you think about this?

`String(format: "%1.0fx%1.0f", screenResolution.width, screenResolution.height)`

This drops the default `.0` which is precision we don't really need.